teachers_digital_platform/css/molecules/form-fields.less

```diff
@@ -0,0 +1,14 @@
+//
+// Custom Form fields styling
+//
+
+.m-form-field__checkbox {
+    .a-checkbox + .a-label.indeterminate {
+        &::before {
+            background-image: url(/static/tdp/img/square-gray.png);
+            background-size: auto 0.875em;
+            background-position: center;
+            background-repeat: no-repeat;
+        }
+    }
+}
```

Comment:
@rrstoll What is this for? There aren’t any checkboxes on the activity detail page.

Reply:
This is search page code, specifically we're changing the facet checkmark to a gray square when only some of a facet parent's child links are active.

Reply:
Originally I had this code within the /cf-enhancements directory and realized it should be moved out of that directory.  The change should have happened in another branch.  Sorry.